bindings/pyroot_experimental/pythonizations/test/rooworkspace.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+import ROOT
+
+class RooWorkspace_test(unittest.TestCase):
+    """
+    Test for the pythonizations of RooWorkspace.
+    """
+
+    # Setup
+    def setUp(self):
+       self.x = ROOT.RooRealVar("x", "x", 1.337, 0, 10)
+       self.ws = ROOT.RooWorkspace("ws", "A workspace")
+
+    # Tear down
+    def tearDown(self):
```

Comment:
This `tearDown` function is not strictly necessary, since before every test function `setUp` will run and override `self.x` and `self.ws`, so the previous values bound to them will be garbage collected as no more references to them will exist. Other than this, looks great!